app/assets/stylesheets/v3/layout_components/forms.scss

```
@@ -223,9 +225,10 @@ input[type="checkbox"] + label,
 .checkbox-label {
   display: inline-block;
   width: 85%;
-  margin-top: 9px;
-  margin-bottom: 9px;
-  margin-left: 38px;
+  margin-top: 10px;
+  margin-bottom: 3px;
+  margin-left: 55px;
+  min-height: 35px;
```

This looks great for single-single line checkboxes but I think it loses some padding on the bottom for multi-line checkboxes (Like on the reg VRAR page)

I think the following would work:
```suggestion
  margin-top: 10px;
  margin-bottom: 10px;
  margin-left: 55px;
```